eventhandler.py

```diff
@@ -11,9 +11,6 @@
     'closed': 'on_pr_closed',
     'labeled': 'on_issue_labeled'
 }
-_test_path_roots = ['a/', 'b/']
-
-DIFF_HEADER_LINE_START = 'diff --git '
```

Comment:
I think I'd prefer to revert this whole commit. The diff-parsing logic isn't something that would need to be overridden during tests and isn't really a property of the API that's being used.